Temporary patch to get clean builds after glib2 update.

Drop this when gnome-settings-daemon is updated to 3.34.0 (once TPNO is approved)

--- a/plugins/housekeeping/gsd-disk-space.c	2019-12-26 16:36:25.165348878 -0800
+++ b/plugins/housekeeping/gsd-disk-space.c	2019-12-26 16:37:45.041244648 -0800
@@ -1017,7 +1017,9 @@
         g_clear_object (&ldsm_monitor);
         g_clear_object (&settings);
         g_clear_object (&privacy_settings);
-        g_clear_pointer (&notification, notify_notification_close);
+        /* NotifyNotification::closed callback will drop reference */
+        if (notification != NULL)
+                notify_notification_close (notification, NULL);
         g_slist_free_full (ignore_paths, g_free);
         ignore_paths = NULL;
 }
